src/Castle.Windsor.Extensions.MsDependencyInjection/Castle.Windsor.Extensions.MsDependencyInjection.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <Import Project="..\..\buildscripts\common.props"></Import>
+
+	<PropertyGroup>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+		<PackageId>Castle.Windsor.Extensions.MsDependencyInjection</PackageId>
```

Comment:
I wasn't aware of that. Besides for .NET Core 3.x it still in separate repo. I wouldn't worry about .NET 5 - knowing how Core teams move, it might still change until they come to release builds

Happy to rename to `Castle.Windsor.Extensions.DependencyInjection`